cluster/gce/windows/README-GCE-Windows-kube-up.md

```diff
@@ -108,87 +111,62 @@ brought up correctly:
+pull the Windows test containers. The smoke test will usually pass on the next
+attempt.
 
-*   TODO(pjh): use patched `cluster/local/util.sh` from
-    https://github.com/pjh/kubernetes/blob/windows-up/cluster/local/util.sh.
+## Running e2e tests against the cluster
 
-*   If necessary run `alias kubectl=client/bin/kubectl` .
+If you brought up an end-to-end test cluster using the steps above then you can
+use the steps below to run K8s e2e tests. These steps are based on
+[kubernetes-sigs/windows-testing](https://github.com/kubernetes-sigs/windows-testing).
 
-*   Set the following environment variables (these values should make sense if
-    you built your cluster using the kube-up steps above):
+*   Build the necessary test binaries. This must be done after every change to
+    test code.
 
     ```
-    export KUBE_HOME=$(pwd)
-    export KUBECONFIG=~/.kube/config
-    export KUBE_MASTER=local
-    export KUBE_MASTER_NAME=kubernetes-master
-    export KUBE_MASTER_IP=$(kubectl get node ${KUBE_MASTER_NAME} -o jsonpath='{.status.addresses[?(@.type=="ExternalIP")].address}')
-    export KUBE_MASTER_URL=https://${KUBE_MASTER_IP}
-    export KUBE_MASTER_PORT=443
+    make WHAT=test/e2e/e2e.test
     ```
 
-*   Download the list of Windows e2e tests:
+*   Set necessary environment variables and fetch the `run-e2e.sh` script:
 
     ```
-    curl https://raw.githubusercontent.com/e2e-win/e2e-win-prow-deployment/master/repo-list.txt -o ${KUBE_HOME}/repo-list.yaml
-    export KUBE_TEST_REPO_LIST=${KUBE_HOME}/repo-list.yaml
-    ```
-
-*   Download and configure the list of tests to exclude:
+    export KUBECONFIG=~/.kube/config
+    export WORKSPACE=$(pwd)
+    export ARTIFACTS=${WORKSPACE}/e2e-artifacts
 
-    ```
-    curl https://raw.githubusercontent.com/e2e-win/e2e-win-prow-deployment/master/exclude_conformance_test.txt -o ${KUBE_HOME}/exclude_conformance_test.txt
-    export EXCLUDED_TESTS=$(cat exclude_conformance_test.txt |
-      tr -d '\r' |                # remove Windows carriage returns
-      tr -s '\n' '|' |            # coalesce newlines into |
-      tr -s ' ' '.' |             # coalesce spaces into .
-      sed -e 's/[]\[()]/\\&/g' |  # escape brackets and parentheses
-      sed -e 's/.$//g')           # remove final | added by tr
+    curl \
+      https://raw.githubusercontent.com/yujuhong/gce-k8s-windows-testing/master/run-e2e.sh \
+      -o ${WORKSPACE}/run-e2e.sh
+    chmod u+x run-e2e.sh
     ```
 
-*   Taint the Linux nodes so that test pods will not land on them:
-
-    ```
-    export LINUX_NODES=$(kubectl get nodes -l beta.kubernetes.io/os=linux,kubernetes.io/hostname!=${KUBE_MASTER_NAME} -o name)
-    export LINUX_NODE_COUNT=$(echo ${LINUX_NODES} | wc -w)
-    for node in $LINUX_NODES; do
-      kubectl taint node $node node-under-test=false:NoSchedule
-    done
-    ```
+    NOTE: `run-e2e.sh` begins with a 5 minute sleep to wait for container images
```

Comment:
Hmm..should we make this configurable?

Reply:
Maybe, or we should just remove it from run-e2e.sh since we're not prepulling containers right now anyway. I guess that's one thing we never ported over from our prototype clusters.